package.json

```diff
@@ -16,6 +16,7 @@
         "build:clear-cache": "rimraf packages/*/tsconfig.tsbuildinfo && rimraf tools/*/tsconfig.tsbuildinfo",
         "build:compare": "tsc --build projects/vrt-compare/tsconfig.json",
         "build:component-inventory": "node ./tasks/build-component-inventory.js",
+        "build:confirm": "lerna exec --ignore \"{@spectrum-web-components/{base,bundle,close-button,clear-button,icons-ui,icons-workflow,iconset,modal,shared,styles},documentation,example-project-rollup,example-project-webpack,swc-templates}\" -- test -f src/index.js",
```

Comment:
This will check for `src/index.js` files in all but the listed packages `base,bundle,close-button,clear-button,icons-ui,icons-workflow,iconset,modal,shared,styles` and then fail is the build artifact requires is missing.